todo_app/data/mongodb.py

```diff
@@ -0,0 +1,87 @@
+import os
+import pymongo
+import datetime
+from bson import ObjectId
+from todo_app.data.item import Item
+
+class MongoDb:
+
+    init = False
+    client = None
+    database = None
+    items_collection = None
+
+    @classmethod
+    def initdb(cls): 
+
+        user=os.getenv('MONGO_USER')
+        if not user:
+            raise ValueError("No MONGO_USER set for application. Please check the enviroment")
+        password=os.getenv('MONGO_USER_PW')
+        if not password:
+            raise ValueError("No MONGO_USER_PW set for application. Please check the enviroment")
+        connection=os.getenv('MONGO_CONNECTION')
+        if not connection:
+            raise ValueError("No MONGO_CONNECTION set for application. Please check the enviroment")
+        server=os.getenv('MONGO_SERVER')
+        if not server:
+            raise ValueError("No MONGO_SERVER set for application. Please check the enviroment")
+        database=os.getenv('MONGO_DATABASE')
+        if not database:
+            raise ValueError("No MONGO_DATABASE set for application. Please check the enviroment")
+
+        cls.client = pymongo.MongoClient(f"{connection}://{user}:{password}@{server}/{database}?w=majority", ssl=True, tlsAllowInvalidCertificates=True)
```

Comment:
Not a problem, but you might find it easier to just ask for a single environment variable (e.g. MONGO_CONNECTION_STRING) that just combines all of these; it will make switching it out easier if the format is subtly different between vendors